features/offer_conditions.feature

```diff
@@ -0,0 +1,64 @@
+@provider
+Feature: Managing conditions on offers
+  Teacher training offers often include conditions that the applicant must fulfil
+  before they can take their place on the course.
+
+  Conditions can be both academic and non-academic. For example, if a candidate
+  still needs to sit exams in order to obtain necessary qualifications or
+  undergo the required criminal record checks to provide evidence of suitability
+  to teach. The provider isn't allowed to make a successful interview a condition
+  (i.e. an interview should happen before and offer is made, not after).
+
+  Background:
+    Given the following providers:
+      | provider code | is an accredited body? |
+      | 10M           | N                      |
+      | U80           | Y                      |
+      | S13           | Y                      |
+    And the following courses:
+      | course code | provider code | accredited body |
+      | X123        | 10M           | U80             |
+
+  Scenario Outline: adding conditions - who can do it and when
+    Providers can add conditions to an application as part of making the offer,
+    or when the application has the 'conditional offer' status. Both the accredited body
+    and the non-accredited body can add conditions.
+
+    When an application has been made to a course X123
+    And the application in "<Application status>" state
+    Then a provider with code "<Provider code>" is able to add conditions: "<Can add conditions?>"
+
+    Examples:
+      | Provider code | Application status   | Can add conditions? | Notes                           |
+      | 10M           | application complete | N                   | Application in the wrong status |
```

Comment:
Should "Application in the wrong status" be either "Application in the wrong state" or "Application has the wrong status"? I think I prefer the first one

Reply:
Agreed – I've updated the wording on #186